app/jobs/inherited_proofing_job.rb

```diff
@@ -0,0 +1,21 @@
+class InheritedProofingJob < ApplicationJob
+  include JobHelpers::StaleJobHelper
+  include Idv::InheritedProofing::ServiceProviderServices
```

Comment:
I don't think you're using anything in this mixin

Reply:
Good catch